taxonomies/ingredients.txt

```diff
@@ -11284,8 +11284,19 @@ wikidata:en:Q1124327
 wikipedia:https://en.wikipedia.org/wiki/Horse_meat
 # ingredient/fr:viande-de-cheval has 21 products in 4 languages @2020-05-21
 
+# description:en:serving of chopped ingredients formed into a disc.
+
+en:patty, patties
+fi:pihvi
+fr:steak
```

Comment:
Is "steak" really used for anything that is not meat?

Reply:
In all speaking French speaking countries, except France, yes. There are "steaks végétaux" that are meat substitutes. But the French legislation restrict this word to meat products.

Reply:
ok